src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");

```

Comment:
The message 'Truck started working' could be more informative. It should indicate what type of Machine is working, which is already done, but it could also describe the work it's doing in more detail. For example, 'Truck is now transporting cargo.'